examples/applications/lt-provider/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT"
 [dependencies]
 chariott-common = { workspace = true }
 chariott-proto = { workspace = true }
-examples-common = { path = "../../common/" }
+examples-common = { workspace = true }
```

Comment:
Is this an unrelated change? If so, why not roll this together with replacing all other paths with the workspace flag?

Reply:
Not strictly unrelated. Since with this PR I needed to take a dependency on `examples-common`, I wanted to do this without causing more duplication. Hence I used workspace inheritance for the dependency that I was going to depend on. To stay consistent, this had to be changed for the projects that already had a dependency on said dependency.